app/src/main/java/com/globant/example/mentorapp/domain/interactor/ListUsersInteractor.java

```diff
@@ -0,0 +1,16 @@
+package com.globant.example.mentorapp.domain.interactor;
+
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.view.fragment.ListUsersInterface;
+
+import retrofit2.Retrofit;
+
+/**
+ * Created by oscar.amezquita on 9/06/2017.
+ */
+
+public interface ListUsersInteractor {
+
+    void getUsersListFromService(SharedUserViewModel model);
```

Comment:
Why you need to pass a model reference? Interactors must not know about ViewModels, they must receive parameters needed to perform operations and return the result